dpe/src/dpe_instance.rs

```diff
@@ -288,6 +416,22 @@ impl TciNodeData {
             tci_current: TciMeasurement([0; DPE_PROFILE.get_tci_size()]),
         }
     }
+
+    fn serialize(&self, dst: &mut [u8]) -> Result<usize, DpeErrorCode> {
```

Comment:
This function could also use something like `bytes::BufMut` (or a manual similar wrapper) to factorize the offset arithmetic. It would be something like:

```rust
fn serialize(&self, mut dst: &mut [u8]) -> Result<usize, DpeErrorCode> {
    let initial_len = dst.len();
    if dst.len() < size_of::<Self>() {
         return Err(...);
    }
    dst.put_u32_le(self.tci_type);
    dst.put(&self.tci_cumulative.0);
    dst.put(&self.tci_current.0);
    Ok(initial_len - dst.remaining_mut())
}